integrations/langchain/src/databricks_langchain/checkpoint.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from databricks.sdk import WorkspaceClient
+from databricks_ai_bridge.lakebase import LakebasePool
+from langgraph.checkpoint.postgres import PostgresSaver
+
+
+class CheckpointSaver(PostgresSaver):
+    """
+    LangGraph PostgresSaver using a Lakebase connection pool.
+
+    instance_name: Name of Lakebase Instance
+    """
+
+    def __init__(
+        self,
+        *,
+        instance_name: str,
+        workspace_client: WorkspaceClient | None = None,
```

Comment:
using `instance_name` for name to be consistent with current Databricks Database SDK, example: https://databricks-sdk-py.readthedocs.io/en/latest/workspace/database/database.html#databricks.sdk.service.database.DatabaseAPI.list_database_catalogs